src/index.ts

```diff
@@ -12,23 +19,29 @@ export const connect = (url) => new Promise((fulfill, reject) => {
     // if there is more than one, the arguments will be joined into a single string, separated by forward slashes '/'
 
     // the inner function should be called with a JSON object, which will be the RPC "params"
-    
+
     // in practice, using this looks like `call("instance", "zome_name", "function_name")({ key: "value" })`
-    
+
     // the whole thing returns a Promise, which will resolve to the result, or an error
     const call = (...method_segments) => (params) => {
-      
+
       const method = method_segments.length === 1 ? method_segments[0] : method_segments.join('/')
       return ws.call(method, params)
     }
     // define a function which will close the websocket connection
     const close = () => ws.close()
-    
-    // fulfill with the Promise with the `call`, and `close` functions, and the raw WebSocket too
-    fulfill({call, close, ws})
+    fulfill({ call, close, ws })
   })
 })
 
-// in case for use in the browser, expose the `connect` function under the holochainClient global variable
-const win = (window as any)
-win.holochainClient = win.holochainClient || {connect}
+function getUrlFromContainer (): Promise<string> {
+  return fetch(CONDUCTOR_CONFIG)
+    .then(data => data.json())
+    .then(json => json.dna_interface.driver.port)
+    .then(port => `ws://localhost:${port}`)
+}
+
+if (typeof(window) !== 'undefined') {
+  const win = (window as any)
+  win.holoclient = win.holoclient || { connect }
```

Comment:
you reverted to holoclient here, but we just moved away from that

Reply:
The lib is now bound to the window as holochainClient